test/functional/pdo_sqlsrv/pdo_test_non_LOB_types.phpt

```diff
@@ -60,19 +64,19 @@ unset($stmt);
 unset($conn);
 
 ?>
---EXPECT--
+--EXPECTREGEX--
 Array
-(
-    [0] => 9223372036854775807
-    [1] => 2147483647
-    [2] => 32767
-    [3] => 255
-    [4] => 1
-    [5] => 9999999999999999999999999999999999999
-    [6] => 922337203685477.5807
-    [7] => 214748.3647
-    [8] => 1.79E+308
-    [9] => 1.1799999E-38
-    [10] => 1968-12-12 16:20:00.000
-    [11] => 
-)
\ No newline at end of file
+\(
+    \[0\] => 9223372036854775807
+    \[1\] => 2147483647
+    \[2\] => 32767
+    \[3\] => 255
+    \[4\] => 1
+    \[5\] => 9999999999999999999999999999999999999
+    \[6\] => 922337203685477.5807

```

Comment:
I think the backslashes are needed here too (and in a few other places) to match a '.'